.gitignore

```diff
@@ -21,3 +21,9 @@ erl_crash.dump
 
 # Ignore package tarball (built via "mix hex.build").
 webengine_kiosk-*.tar
+
+# QT Creator config
+/src/kiosk.pro.user
+
+# IDE directory
+.elixir_ls
```

Comment:
```suggestion
```

The rule of thumb is to never commit IDE files to `.gitignore` files in OSS projects, since then you need to add lines for all editors and it's impossible to maintain these. Luckily, there's a better way and that's to use a global `.gitignore` file. See https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Reply:
I will fix this before merge